### ISSUE TITLE
Support AppVeyor Ubuntu images

### ISSUE THREAD

**Issue by axelheer** [codecov/codecov-bash#125]
According to the [official documentation](https://www.appveyor.com/docs/environment-variables/) the environment variables `CI` and `APPVEYOR` don't contain the values `True` but `true` (lower case!).

Thus, the bash script is unable to detect that. I'd just copy the [code block](https://github.com/codecov/codecov-bash/blob/4678d212cce2078bbaaf5027af0c0dafaad6a095/codecov#L663-L673) for AppVeyor, since I'm not at all into bash scripting. So, I'm filing an issue instead. 🤓

**Comment by FantasticFiasco** [comment]
Another solution until we have a fix is to set the environment variables our self.

```bash
export CI="True"
export APPVEYOR="True"
bash <(curl -s https://codecov.io/bash)
```

**Comment by drazisil** [comment]
Resolved by #138
